sunpy/util/config.py

```diff
@@ -167,6 +174,15 @@ def copy_default_config(overwrite=False):
     if not _is_writable_dir(user_config_dir):
         raise RuntimeError(f'Could not write to SunPy config directory {user_config_dir}')
 
+    config_path = Path(dirs.site_config_dir)
+    site_config_file = config_path.joinpath(config_filename)
+    if site_config_file.exists():
+        message = f"Site config file {str(site_config_file)} exists. " \
+            "You must update the User config file to take the " \
+            "site configuration into account manually. Consult your" \
+            "site manager."
```

Comment:
```suggestion
        message = (f"Site config file {str(site_config_file)} exists. " 
            "You must update the User config file to take the " 
            "site configuration into account manually. Consult your" 
            "site manager."
            )
```
That should(?) work.